concrete/src/Updater/Migrations/Migrations/Version20160725000000.php

```diff
@@ -367,7 +367,7 @@ protected function importAttributeKeys()
             $this->importAttributeKeySettings($row['atID'], $row['akID']);
             switch ($akCategory) {
                 case 'pagekey':
-                    $rb = $this->connection->executeQuery('select * from CollectionAttributeValues where akID = ? group by avID', [$row['akID']]);
+                    $rb = $this->connection->executeQuery('select DISTINCT(avID) from CollectionAttributeValues where akID = ?', [$row['akID']]);
```

Comment:
I think that `select distinct avID from CollectionAttributeValues where akID = ?` would be more standard (I never found a documentation about using `distinct()` as a function)

Reply:
You are right. This is a discussion of style though. The parentheses are optional and are not linked to `DISTINCT` as some sort of function. This is a personal quirk of mine.

In another query `DISTINCT(avID, atID)` would be equivalent  to `DISTINCT avID, atID`, `DISTINCT (avID), atID` , `DISTINCT avID, (atID)` , and `DISTINCT (avID), (atID)`

Reply:
What about `SELECT DISTINCT avID, ((((((((((atID))))))))))` ? Just joking :wink: